addons/pos_restaurant/static/tests/tours/floor_screen_tour.js

```diff
@@ -176,6 +176,7 @@ registry.category("web_tour.tours").add("TableMergeUnmergeTour", {
             ProductScreen.clickPayButton(),
             PaymentScreen.clickPaymentMethod("Cash"),
             PaymentScreen.clickValidate(),
+            Dialog.discard(),
```

Comment:
I think it's better to specify a specific dialog information here. A simple check on the content is enough. Do it for all the other instances.